lib/cb/models/implementations/job.rb

```diff
@@ -20,7 +20,7 @@ class Job < ApiResponseModel
                     :location, :distance, :latitude, :longitude, :location_formatted, :location_metro_city,
                     :description, :requirements, :employment_type,
                     :details_url, :service_url, :similar_jobs_url, :apply_url,
-                    :begin_date, :end_date, :posted_date, :posted_time,
+                    :begin_date, :end_date, :posted_date, :posted_time, :posting_date,
```

Comment:
I am not sure why they are returned differently. :disappointed: But one of the job search API and the other is the recs. 
The posted_date is returned by the job search API and assigned as so:
````
@posted_date                  = args['PostedDate'] || ''
@posted_time                  = args['PostedTime'] || ''
````

The posting_date is returned by the recs api:
````
"PostingDate": "2/19/2016 12:00:00 AM",
````